tests/foreman/cli/test_remoteexecution.py

```diff
@@ -94,6 +94,12 @@ def fixture_enable_receptor_repos(request, default_sat):
     default_sat.enable_repo(constants.REPOS['rhs7']['id'])
 
 
+@pytest.fixture()
+def infra_host(request, default_sat, capsule_configured):
```

Comment:
Will any future tests be using this fixture? Should we move into pytest_fixtures?

Reply:
hm, seems unlikely right now